content/applications/general/email_communication/email_servers.rst

```diff
@@ -197,26 +197,21 @@ alias in your mail server.
 Set up different dedicated servers for transactional and mass mails
 ===================================================================
 
-Odoo is subject to a :ref:`daily email limit <email_communication/daily_limit_mail>` to prevent abuse. 
-However, if needed, you can use a separate Mail Transfer Agent (MTA) servers for transactional
-e-mails and mass mailings.
-Example: use Odoo's own mail server for transactional e-mails, and Sendgrid, Amazon SES, or Mailgun
-for mass mailings. Another alternative is to use Postmark for transactional e-mails, and Amazon SES
-or Sendgrid for mass mailings.
+In Odoo a separate Mail Transfer Agent (MTA) server can be used for transactional e-mails and mass mailings.
+Example: Use Postmark or SendinBlue for transactional e-mails, and Amazon SES, Mailgun or Sendgrid for mass mailings.
 
 .. note::
    A default outgoing email server is already configured. You should not create an alternative one
    unless you want to use a specific external outgoing email server for technical reasons.
 
 To do this, you should first activate the :ref:`developer mode <developer-mode>` and then go to
 :menuselection:`Settings --> Technical --> Outgoing` e-mail servers. There you have to create two
-e-mail MTA server settings. One for transactional e-mails and one for mass mail servers. Be sure to
-mark the priority of transactional e-mail servers as low as the mass email servers.
+e-mail MTA server settings; one for the transactional e-mails and one for the mass mailing server. 
+Make sure to give priority to the transactional server over the mass mailing server by providing 
+a lower priority number for the transactional MTA server.
 
 Now, go to :menuselection:`Email Marketing --> Settings` and enable *Dedicated Server*.
 With these settings, Odoo uses the server with the lower priority for transactional emails, and the
 server here selected for mass mails.
 Note that in this case, you have to set your domain's Sender Policy Framework (SPF) records to
```

Comment:
Replace `you have to set your domain's Sender Policy Framework (SPF) records` with `the domain's Sender Policy Framework (SPF) records must be set `